db/migrate/20240506184235_add_signature_fields_flag_to_organizations.rb

```diff
@@ -0,0 +1,11 @@
+class AddSignatureFieldsFlagToOrganizations < ActiveRecord::Migration[7.0]
+  def up
+    add_column :organizations, :include_signature_fields_on_distribution_printout, :boolean
```

Comment:
Can we make this a bit less wordy? Maybe `signature_for_distribution_pdf`?

Reply:
surely, naming things is always a pain so I tend to default to extra verbose

Reply:
I've gone ahead and made this change.